src/runtime/runtime.js

```diff
@@ -888,6 +903,7 @@ function createPublicRuntime(runtime) {
     attachSmartButton: runtime.attachSmartButton.bind(runtime),
     getPropensityModule: runtime
         .getPropensityModule.bind(runtime),
+    eventManager: runtime.eventManager.bind(runtime),
```

Comment:
Just noticed this change when reviewing the other change. Is there any reason to expose this outside of SwG? Looks like AMP is already able to send in the event manager to ConfiguredRuntime and we create one internally if required. I suggest removing line 906

Reply:
If you say it is not necessary, I believe you.  I'm not sure why those are there it just seemed to fit along with everything else quite well.  Removed.